pjsip/src/pjsip/sip_dialog.c

```diff
@@ -1630,6 +1630,15 @@ PJ_DEF(pj_status_t) pjsip_dlg_send_response( pjsip_dialog *dlg,
         pj_assert(status == PJ_SUCCESS);
     }
 
+    /* Copy the initial destination host to tdata. This information can be
+     * used later by transport for transport selection.
+     */
+    if (dlg->initial_dest.slen) {
+        pj_strdup(tdata->pool, &tdata->dest_info.name, &dlg->initial_dest);
+        PJ_LOG(5, (THIS_FILE, "Setting initial dest %.*s",
+            (int)dlg->initial_dest.slen, dlg->initial_dest.ptr));
+    }
+
```

Comment:
- Why not set in the creation of the tdata response (just like in request scenario)?
- A tdata response can be sent multiple times (100, 18x, 2xx), this will strdup() multiple times?

Reply:
- oh okay, the clone does not clone the `dest_info` field indeed.
- do you think adding check, whether `dest_info` is already set (by app for some reason), may be safer?

Reply:
Yes I agree adding a check will be safer